python/nrel/routee/compass/io/generate_dataset.py

```diff
@@ -73,26 +73,26 @@ def generate_compass_dataset(
     agg = agg if agg is not None else np.mean
 
     # pre-process the graph
-    log.info("processing graph topology and speeds")
+    print("processing graph topology and speeds")
```

Comment:
but what about logging? maybe we just need to be sure to set up logging.basicConfig correctly in the python library?

Reply:
yeah there's probably the _right_ way to do it that I didn't have the time to investigate; the motivation was that it was not logging to the jupyter notebook and so it was hard to tell what was happening.